tmt.spec

```diff
@@ -48,6 +48,7 @@ BuildRequires: python%{python3_pkgversion}-mock
 BuildRequires: python%{python3_pkgversion}-requests
 BuildRequires: python%{python3_pkgversion}-testcloud
 BuildRequires: python%{python3_pkgversion}-markdown
+BuildRequires: python%{python3_pkgversion}-junit_xml
```

Comment:
```suggestion
BuildRequires: python%{python3_pkgversion}-junit
```

What about using just `junit` for the plugin name? I see this is used below as well.

Reply:
This is a buildrequire for PyPi

Reply:
Ah, my bad, seems I was tired ;-)